### DISCUSSION TITLE
GO2RTC CPU usage

### DISCUSSION THREAD

**Discussion by Chrissica06** [blakeblackshear/frigate#15491] [Ask A Question] [answered]
Does Go2rtc use the GPU or CPU?   Not that usage is high, but it is running higher then GPU usage
![image](https://github.com/user-attachments/assets/062c3a89-0c63-4740-b8b7-38825140ab05)

**Comment by NickM-27** [comment, marked as answer]
It can use either depending on what it's doing but the act of restreaming is a cpu task

**Reply by Chrissica06** [reply]
Thanks just wondering.
